File: UIPreparation/UI/src/app/core/components/admin/storage/models/storageDto.ts
export class StorageDto {
    id?:number; 
    createdUserId?:number; 
    userName?:string;
    productName?:string;
    createdDate?:(Date | any); 
    lastUpdatedUserId?:number; 
    lastUpdatedDate?:(Date | any); 
    status:boolean; 
    isDeleted:boolean; 
    productId?:number; 
    unitsInStock?:number; 
    isReady:boolean; 
}

File: Entities/Dtos/StorageDto.cs
using Core.Entities;
using Entities.Concrete;
using Nest;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.Dtos
{
    public class StorageDto: Concrete.BaseEntity, IDto
    {
        public int ProductId { get; set; }
        public string ProductName { get; set; }
        public string UserName { get; set; }
        public int UnitsInStock { get; set; }

        public bool IsReady { get; set; }

        public string Size { get; set; }
    }
}
